src/components/Integrations/BigQuery/AuthorizationPage.js

```diff
@@ -59,23 +60,118 @@ const AuthorizationLayout = ({ dependencies: { bigQueryClient, dopplerUserApiCli
       <Promotional
         title={_('big_query.free_title')}
         description={_('big_query.free_text_summary')}
-        features={[_('big_query.free_ul_item_insights'), _('big_query.free_ul_item_filter')]}
+        features={[
+          _('big_query.free_ul_item_strategy'),
+          _('big_query.free_ul_item_insights'),
+          _('big_query.free_ul_item_filter'),
+        ]}
         paragraph={_('big_query.free_text_strong')}
         actionText={_('big_query.free_btn_redirect').toUpperCase()}
         actionUrl={_('big_query.upgrade_plan_url')}
         logoUrl={bigQueryLogo}
         previewUrl={dataStudioGif}
+        caption={<FormattedMessageMarkdown id="big_query.free_text_data_studio_MD" />}
       />
     );
   }
-
   return (
-    <div className="dp-container">
-      <div className="dp-rowflex p-t-48">
-        <div className="col-lg-5">
-          <AuthorizationForm emails={data} onSubmit={onSubmit} />
+    <div data-active-menu="35" className="dp-library">
+      <div className="dp-container">
+        <div className="dp-rowflex">
+          <div className="col-sm-12 m-t-24">
+            <header className="m-b-30">
+              <a
+                href="index.html"
+                className="dp-button button-medium secondary-green button--round"
+              >
+                {' '}
+                Back{' '}
```

Comment:
```suggestion
{_('common.back')}
```

Reply:
this code was removed because is unneccesary, the back button only used for UI library purpose